src/Illuminate/Collections/Collection.php

```diff
@@ -943,9 +953,19 @@ public function search($value, $strict = false)
      *
      * @return mixed
      */
-    public function shift()
+    public function shift($number = 1)
```

Comment:
You should also update the typehint to accept an `int`.

Reply:
Good point. I've updated the docblocks to reflect the parameter.